a3p-integration/proposals/a:upgrade-next/upgradeVaults.test.js

```diff
@@ -92,17 +92,38 @@ test.serial('open a marginal vault', async t => {
     activeVaultsAfter.length > currentVaults.length,
     `vaults count should increase, ${activeVaultsAfter.length}, ${currentVaults.length}`,
   );
-});
+};
 
-test.serial('trigger auction', async t => {
+const triggerAuction = async t => {
   await pushPrices(5.2, 'ATOM', oraclesByBrand);
 
   const atomOut = await getPriceQuote('ATOM');
   t.is(atomOut, '+5200000');
-});
+};
 
-test('new auction vat', async t => {
+const makeNewAuctionVat = async t => {
   const details = await getDetailsMatchingVats('auctioneer');
   // This query matches both the auction and its governor, so double the count
   t.true(Object.keys(details).length > 2);
+};
+
+// test.serial() isn't guaranteed to run tests in order, so we cobble together a driver
+test('driver', async t => {
```

Comment:
this module has helpers functions, but not a "driver" object.

it's also not testing the driver.
```suggestion
test('auction after upgrade', async t => {
```

Reply:
renamed